docs/blog/_posts/2020-08-31-27th-dotty-milestone-release.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.27.0-RC1 - ScalaJS, performance, stability
+author: Anatolii Kmetiuk
+authorImg: /images/anatolii.png
+date: 2020-08-31
+---
+
+Hello! We are excited to announce 0.27.0-RC1 of Dotty. In this version, we bring ScalaJS support to Dotty. As we are getting closer to the Scala 3 release, we continue shifting our focus to stability and performance, which are the central theme of this release.
+
+You can try out this version right now, from the comfort of your SBT, by visiting the [home page](https://dotty.epfl.ch/) and scrolling down to the "Create a Dotty Project" section.
+
+Alternatively, you can try this version of Scala online via [Scastie](https://scastie.scala-lang.org/). Once you're there, click "Build Settings" and set "Target" to "Dotty".
+
+Enjoy the ride🚀!
+
+<!--more-->
+# Welcome ScalaJS support
+PR [#9637](https://github.com/lampepfl/dotty/pull/9637) brings ScalaJS support to Dotty. Now, if you use the ScalaJSPlugin in your Dotty SBT project, the project will be correctly compiled to compile to JavaScript.
```

Comment:
That's probably a bit terse, given that there were many PRs that brought that support (#9637 was only the final straw) and that there are still limitations (no non-native JS classes and no exports). I suggest the following as a more complete description of what is and isn't supported:
```markdown
# Welcome Scala.js support (with caveats)

This release brings enough support for Scala.js that it should be actually usable in a number of projects.
To use it, make sure of the following:

* Use sbt-scalajs v1.1.1 or later
* Use sbt-dotty v0.4.2 or later (TODO v0.4.3???)
* Use `scalaVersion := "0.27.0-RC1"` or later for Dotty
* Use `enablePlugins(ScalaJSPlugin)` or a `crossProject`

When the above are combined, sbt-scalajs ans sbt-dotty will coordinate to transparently configure your project with Dotty with its Scala.js support.

In this release, the following features are supported:

* Dependencies with `%%%`, including with `withDottyCompat`,
* The entire portable subset of the language, i.e., code that compiles both on the JVM and on JS,
* Calling JavaScript APIs, including those defined in dependencies.

The following features are *not supported yet*:

* Define non-native JS classes (i.e., classes extending `js.Any` but without `@js.native`): they will report compile errors
* Exports of all kinds (i.e., `@JSExportXYZ`): they will be silently ignored

To the best of our knowledge, cross-compiling libraries should be able to use Scala.js with Dotty in plain capacity.
If you experience a bug with anything except the unsupported features mentioned above, please file a bug report.
```
Note the ??? about the version of sbt-dotty. I think the publishing of sbt-dotty v0.4.2 never happened because of https://github.com/lampepfl/dotty/pull/9654, so I don't know if we'll republish sbt-dotty v0.4.2 or v0.4.3.

Reply:
> Note the ??? about the version of sbt-dotty. I think the publishing of sbt-dotty v0.4.2 never happened because of #9654, so I don't know if we'll republish sbt-dotty v0.4.2 or v0.4.3.

0.4.2 appears to have been published successfully: https://github.com/lampepfl/dotty/actions/runs/227351085